src/dsymbol/tests.d

```diff
@@ -484,6 +484,60 @@ unittest
 	assert(a.data.ptr == b.data.ptr);
 }
 
+// this is for testing if breadcrumbs are properly generating the calltip
+// we only process symbols with suffixes (ptr, array, aa)
+unittest
+{
+	ModuleCache cache = ModuleCache(theAllocator);
+
+	writeln("Running calltip tests...");
+	auto source = q{ 
+		struct A {int a;}
+		alias APTR = A*;
+
+		A value; A* ptr = &value; A** ptrptr = &ptr; A[] arr; A*[] arrptr; A[int] map; A*[int] mapptr;
```

Comment:
can you add an `A*[][int][]*` test case here?

Reply:
Done!